package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "new-react-app",
+  "version": "1.0.0",
+  "description": "A new react app without CRA",
+  "main": "./src/index.tsx",
+  "engines": {
+    "node": "16.15.1",
+    "npm": "8.11.0"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.prod.js",
+    "dev": "webpack-dev-server --hot --config webpack.dev.js --port 3000",
+    "start": "npm run build & serve -s build -p 3010",
+    "test": "test"
```

Comment:
remove the test command for now, as we've not yet configured any automated testing feature.

Reply:
removed